samples/rag-aisearch/csharp-inproc/SemanticAISearchEmbeddings.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.2.0" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.3.0" />
```

Reply:
my bad, this version always come up with default template on VS and forget to update.